xmrstak/misc/executor.cpp

```diff
@@ -487,26 +488,8 @@ void disable_sigpipe()
 inline void disable_sigpipe() {}
 #endif
 
-void executor::ex_main()
-{
-	disable_sigpipe();
-
-	assert(1000 % iTickTime == 0);
-
-	xmrstak::miner_work oWork = xmrstak::miner_work();
-
-	// \todo collect all backend threads
-	pvThreads = xmrstak::BackendConnector::thread_starter(oWork);
-
-	if(pvThreads->size()==0)
-	{
-		printer::inst()->print_msg(L1, "ERROR: No miner backend enabled.");
-		win_exit();
-	}
-
-	telem = new xmrstak::telemetry(pvThreads->size());
-
-	set_timestamp();
+void executor::init_pools(void)
```

Comment:
there is not need for the explicit `void`

Reply:
I have removed explicit void, however coming from embedded development for mission critical apps i always put void in the func that is not supposed to take any args, but this is in C :)